src/common/error.rs

```diff
@@ -290,16 +290,16 @@ where
     where
         C: Into<String>,
     {
-        self.map_err(|e| e.into().add_context(ctx))
+        self.map_err(|err| err.into().add_context(ctx))
```

Comment:
Which clippy lint did this violate? Does it check an identifier with single letter?